<launch>
    <node 
        name="hand_setpoint_node" 
        pkg="manipulator_controller_cpp" 
        type="hand_setpoint_node" 
        respawn="true"
        output="screen">
        <rosparam command="load" file="$(find manipulator_controller_cpp)/config/parameters.yaml" />
    </node>

    <node 
        name="hand_setpoint_publisher_node" 
        pkg="manipulator_controller_cpp" 
        type="hand_setpoint_publisher_node" 
        respawn="true"
        output="screen">
    </node>

    <!-- <node 
        name="body_setpoint_node"
        pkg="manipulator_controller_cpp" 
        type="body_setpoint_node" 
        respawn="true"
        output="screen">
    </node> -->

</launch>